lib/Dialect/PolyExt/IR/PolyExtDialect.cpp

```diff
@@ -0,0 +1,63 @@
+#include "include/Dialect/PolyExt/IR/PolyExtDialect.h"
+
+#include "include/Dialect/Poly/IR/PolyTypes.h"
+#include "include/Dialect/Poly/IR/Polynomial.h"
+#include "include/Dialect/PolyExt/IR/PolyExtOps.h"
+#include "llvm/include/llvm/ADT/TypeSwitch.h"            // from @llvm-project
+#include "mlir/include/mlir/IR/Builders.h"               // from @llvm-project
+#include "mlir/include/mlir/IR/DialectImplementation.h"  // from @llvm-project
+
+// Generated definitions.
+#include "include/Dialect/PolyExt/IR/PolyExtDialect.cpp.inc"
+#define GET_OP_CLASSES
+#include "include/Dialect/PolyExt/IR/PolyExtOps.cpp.inc"
+
+namespace mlir::heir::poly_ext {
+
+//===----------------------------------------------------------------------===//
+// EncryptedArith dialect.
```

Comment:
```suggestion
// PolyExt dialect.
```

Or possible just remove the ASCII style comment!

Reply:
+1 the ascii style comments don't do anything for me